internal/exec/stages/disks/luks.go

```diff
@@ -531,3 +566,135 @@ func (d LuksDump) hasFlag(flag string) bool {
 	}
 	return false
 }
+
+// collect the cex card domains xx.xxxx,
+func getAllCexOnlineApqns() (string, error) {
```

Comment:
is Apqns an acronym? if so can we cap it i.e APQNS

Reply:
it stands for Adjunct Processor Queue Number. An APQN designates the combination of a cryptographic coprocessor (adapter) and a domain. I'll  change to  Caps.